docs/notebooks/at_with_nips.ipynb

```diff
@@ -0,0 +1,2162 @@
+{
```

Comment:
please add cells with markdown. the first cell should explain the contents of the tutorial.
please add cells that show the link author-topic-topic_description